api/organisations/admin.py

```diff
@@ -132,12 +145,13 @@ def get_queryset(self, request):  # type: ignore[no-untyped-def] # pragma: no co
             )
             .all()
         )
+        return queryset
 
-    def num_users(self, instance: Organisation) -> int:
-        return instance.num_users  # type: ignore[no-any-return]
+    def num_users(self, instance: OrganisationWithAnnotations) -> int:
+        return instance.num_users
 
-    def num_projects(self, instance: Organisation) -> int:
-        return instance.num_projects  # type: ignore[no-any-return]
+    def num_projects(self, instance: OrganisationWithAnnotations) -> int:
+        return instance.num_projects
```

Comment:
In order to solve the coverage errors here, we can just add `# pragma: no cover` comments to the methods.